client/src/cordova/build.action.mjs

```diff
@@ -142,7 +169,12 @@ const JAVA_BUNDLETOOL_VERSION = '1.8.2';
 const JAVA_BUNDLETOOL_RESOURCE_URL = `https://github.com/google/bundletool/releases/download/1.8.2/bundletool-all-${JAVA_BUNDLETOOL_VERSION}.jar`;
 
 async function androidRelease(ksPassword, ksContents, javaPath, verbose) {
-  const androidBuildPath = path.resolve(getRootDir(), 'platforms', 'android');
+  const androidBuildPath = path.resolve(
+    getRootDir(),
+    'client',
```

Comment:
Here is the only change to this file (other changes are just to fix the lint errors)